cmd/openshift-install/create.go

```diff
@@ -96,10 +96,11 @@ var (
 
 				err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 				if err != nil {
+					logrus.Error("Installation failure detected. Running gather bootstrap...")
```

Comment:
I don't like `logrus.Error` for "here's our plan".  I'm fine with two info-level logs inside `runGatherBootstrapCmd` if you feel a need to separate SSH config from the SSH connection.  And I'm fine with an error-level log here that includes `err`.  And I'm fine with nothing here, because if the coming gather succeeds you won't have two separate errors for users to be confused about.

Reply:
> I don't like logrus.Error for "here's our plan". 
> And I'm fine with nothing here, because if the coming gather succeeds you won't have two separate errors for users to be confused about.
It sounds like you have changed your mind about this https://github.com/openshift/installer/pull/2244#discussion_r315978679 or perhaps I am misinterpreting? To me this current implementation seems inline with ttps://github.com/openshift/installer/pull/2244#issuecomment-523242454

I don't feel like there is a clear direction here or that we are all going to reach an agreement, which is fine because I think this is subjective and difficult. I will revert this change and try to make a more minor change which will hopefully have at least some improvement.

Reply:
> It sounds like you have changed your mind about this...

I'm still fine with an error-level log here as long as it includes `err`.  But I'm fine with 2992564e0 too.